test/files/00T_ideal.pdb

```diff
@@ -0,0 +1,45 @@
+ATOM      1  N19 00T A   1      -2.126  -2.241  -0.764  1.00 10.00           N
+ATOM      2  C10 00T A   1      -1.652  -1.628   0.484  1.00 10.00           C
+ATOM      3  C6  00T A   1      -0.631  -0.566   0.166  1.00 10.00           C
+ATOM      4  C5  00T A   1      -1.043   0.686  -0.251  1.00 10.00           C
+ATOM      5  C32 00T A   1      -2.513   0.988  -0.389  1.00 10.00           C
+ATOM      6  N33 00T A   1      -3.078   1.283   0.934  1.00 10.00           N
+ATOM      7  C4  00T A   1      -0.107   1.661  -0.542  1.00 10.00           C
+ATOM      8  C3  00T A   1       1.241   1.384  -0.416  1.00 10.00           C
+ATOM      9  C2  00T A   1       1.654   0.131   0.002  1.00 10.00           C
+ATOM     10 CL1  00T A   1       3.348  -0.217   0.160  1.00 10.00          CL
+ATOM     11  C7  00T A   1       0.717  -0.842   0.298  1.00 10.00           C
+ATOM     12 HN19 00T A   1      -1.355  -2.609  -1.301  1.00 10.00           H
```

Comment:
Not that there's an issue, but we might want to check some of these H atoms. In older or non-standard PDB files where the right-most column isn't present, you're supposed to infer from HN19, and of course "Hn" isn't an element. So I'd add atoms 12, 16, and 17 to the test list.